browser/resources/settings/brave_sync_page/brave_sync_page.ts

```diff
@@ -58,14 +58,18 @@ export class SettingsBraveSyncPageElement extends SettingsBraveSyncPageElementBa
     };
   }
 
-  private syncStatus_: SyncStatus;
+  private syncStatus_: BraveSyncStatus;
   private isEncryptionSet_: boolean;
   private syncLabel_: string;
 
   browserProxy_: SyncBrowserProxy = SyncBrowserProxyImpl.getInstance();
   braveBrowserProxy_: BraveSyncBrowserProxy = BraveSyncBrowserProxy.getInstance();
 
   computeSyncLabel_() {
+    if (this.syncStatus_ !== undefined &&
```

Comment:
The dependency list for `computeSyncLabel_` needs to be updated to contain anything we're checking in this function so that it re-computes when it changes. Right now it is simply `computeSyncLabel_(syncStatus_.firstSetupInProgress)` and looks like it needs to change to `computeSyncLabel_(syncStatus)`

Reply:
thanks, fixed at https://github.com/brave/brave-core/pull/16117/commits/9a490dc2a9d65ee0a3c6459fcbfb593326dc8d07